src/components-examples/material/table/table-row-context/table-row-context-example.html

```diff
@@ -1,5 +1,5 @@
 <table mat-table [dataSource]="data" class="mat-elevation-z8">
-  <!-- Index Column -->
+  <!-- Implicit Column -->
```

Comment:
This one should stay as "Index"

Reply:
But already exists the index column, do you can see in the images. Implicity is another column.
![image](https://user-images.githubusercontent.com/31555105/75718909-8d227080-5cb2-11ea-82ae-57407c2ded61.png)
![image](https://user-images.githubusercontent.com/31555105/75718973-af1bf300-5cb2-11ea-86a4-c751f3a4d6aa.png)

Reply:
Ah, I thought that `$implicit` was index here, but I was mistaken